src/libstore/worker-protocol-connection.cc

```diff
@@ -69,13 +69,13 @@ WorkerProto::BasicClientConnection::processStderrReturn(Sink * sink, Source * so
             } else {
                 auto error = readString(from);
                 unsigned int status = readInt(from);
-                ex = std::make_exception_ptr(Error(status, error));
+                ex = std::make_exception_ptr(Error(status, "[daemon] " + error));
             }
             break;
         }
 
         else if (msg == STDERR_NEXT)
-            printError(chomp(readString(from)));
+            printError("[daemon] " + chomp(readString(from)));
```

Comment:
Prefixing every daemon message with `[daemon]` seems very annoying. Generally the use of the daemon is supposed to be transparent, so we don't want to spam the user with gazillions of `[daemon]` message. There are a few cases where it helps to disambiguate where the error originated, but most of the time we don't care.

Reply:
This was for debugging; forgot to remove. Will do. I think a setting would be useful.